### ISSUE TITLE
How to predict label and θ when testing

### ISSUE THREAD

**Issue by c1a1o1** [elvisyjlin/AttGAN-PyTorch#12]
How to predict label and θ when testing

**Comment by elvisyjlin** [comment]
To predict labels from D, you can achieve it by
```python
import torch
_, logits = attgan.D(img)
predictions = torch.sigmoid(logits)
```
As for the θ, _Extension for Attribute Style Manipulation_ is not implemented. You will find no θ here.

However, in my implementation, only weights of G is saved. It is not feasible to predict labels from my pretrained weights. You have to change [this line](https://github.com/elvisyjlin/AttGAN-PyTorch/blob/master/train.py#L172) from `saveG()` to `save()` and train a model yourself to get a good D.

**Comment by c1a1o1** [comment]
Thank  you very much!
